src/lib/i18n.ts

```diff
@@ -10,7 +10,18 @@ i18next
   .init({
     nsSeparator: false,
     keySeparator: false,
-    fallbackLng: 'en',
+    // Since these languages have different translation folder names
```

Comment:
This code doesn't look wrong, but I don't think it's necessary. Even without this configuration, if the user's language is `fr-fr`, i18next still seems to normalize the casing and requests `fr-FR/translations.json`. Have you checked that it is broken without this configuration?

This configuration still seems useful to keep though since it does more than just fix the casing: `'fr-ca': ['fr-CA', 'fr-FR', 'en'],`

Reply:
I thought I did, but let me double check

Reply:
You are correct. I think I tested it without restarting my local, which affected it.